pkg/backup/util/util.go

```diff
@@ -54,11 +64,6 @@ func FilterAndSortBackups(names []string) []string {
 		if !IsBackup(n) {
 			continue
 		}
-		_, err := GetRev(n)
```

Comment:
why is removed?

Reply:
This check is unnecessary at the hindsight.

Reply:
Actually some test is relying on this..
Added back